### DISCUSSION TITLE
复制评论链接如何使用？

### DISCUSSION THREAD

**Discussion by Kal-tsit63** [the1812/Bilibili-Evolved#3443] [提问 / Q&A]
![image](https://user-images.githubusercontent.com/85188780/174493745-4f942fca-2129-4083-8268-997b89c644a9.png)
根据插件描述，是在评论菜单处可以复制  但本人属实找不到评论菜单在哪，想请教一下各位有没有在用的？如何使用？

**Comment by Geeyun-JY3** [comment]
点击评论右边的︙

**Reply by Kal-tsit63** [reply]
> 点击评论右边的︙

我点击以后没有出现复制链接的选项,只有默认的加入黑名单和举报

**Comment by Kal-tsit63** [comment]
![image](https://user-images.githubusercontent.com/85188780/174494695-39cfa202-78f0-4cb8-8709-86bc5d233431.png)

**Reply by Geeyun-JY3** [reply]
看置顶，新版播放页未适配：#2615
